tawazi/config.py

```diff
@@ -31,6 +31,10 @@ class Config(BaseSettings):
     # choose the default Resource to use to execute the ExecNodes
     TAWAZI_DEFAULT_RESOURCE = Resource.thread
 
+    # choose wether to use DILL or pickle for serializing ExecNodes
+    #  This should be set to True if user wants to use Resource=Process!
+    TAWAZI_DILL_NODES = False
```

Comment:
Do you plan on doing a validator for this variable in the config to dynamically set it to true if Resource=Process ?